### DISCUSSION TITLE
Collegamento Shelly 2.5 e pulsanti con neutro interrotto.

### DISCUSSION THREAD

**Discussion by cl-7k-01** [indomus/forum#1380] [Hardware & Firmware]
Ciao a tutti,
ho un dubbio da esporvi e vorrei un parere da chi ne sa più di me.
In casa avevo due punti luce comandati da pulsanti (3 pulsanti per punto luce) e relè (Vimar, mod. 3991 QUID).
Il collegamento era fatto come da figura 1 (propongo in schema un solo punto luce per sintesi) con il neutro che va ai pulsanti.
Ho avuto l'idea di installare uno shelly 2.5 per domotizzare i due punti luce e, inconsciamente ed incautamente ho installato lo shelly come da figura 2.5 sebbene in tutte i schemi di collegamento che trovo non venga mai portato il neutro ai pulsanti, ma la sola fase.
Lo Shelly sta funzionando piuttosto bene e non mi ha dato alcun problema anche se collegato in maniera diversa da quella indicata in tutte le istruzioni/manuali ma mi chiedo: sto rischiando qualcosa? è necessario che corregga l'errore togliendo il neutro e portando ai pulsanti la fase? che ne pensate?
Grazie in anticipo
![Figura01](https://user-images.githubusercontent.com/4005789/150695144-f769a4b1-ca28-4f8e-8291-4c6aed0d9d91.jpg)
![Figura02](https://user-images.githubusercontent.com/4005789/150695149-e5e868e8-8316-4947-9cb0-4c47ef20cbd6.jpg)

**Comment by dante70** [comment]
ciao,rimango perplesso per il fatto che stai dicendo che funziona bene.
io ho una decina di shelly e anche diversi sonoff dual r3 che sono gli equivalenti piu' o meno di altra marca e ho sempre rispettato gli schemi. Ho sempre pensato che venisse preso come riferimento il neutro e quando sull'ingresso(SW1 e SW2) ti arrivava la fase  veniva recepito come cambio di stato.Mi sfugge come possa funzionare ma lascerei cosi' visto che va

**Comment by laspinacristian** [comment]
sugli input (sw1 / sw2) di tutti gli shelly puoi portare indifferentemente sia fase che neutro. Il funzionamento non cambia.
Lo shelly, una volta alimentato a 230ac (non saprei quando alimentato in dc) mette una tensione in uscita di circa 110v sul morsetto sw. Quando viene rilevata una differenza di potenziale sia che questa sia in positivo (fase +230) che in negativo (neutro 0v) lo shelly rileva l'input. Potenzialmente potresti attivare l'input anche collegando la messa a terra.

**Reply by dante70** [reply]
Grazie dell'informazione.
in casa shelly ci hanno visto lungo .......hanno pensato anche a chi interrompe il neutro e non la fase

**Reply by cl-7k-01** [reply]
> Grazie dell'informazione. in casa shelly ci hanno visto lungo .......hanno pensato anche a chi interrompe il neutro e non la fase

Oppure hanno pensato che il loro prodotto spesso sarebbe stato usato per sostituire relè tradizionali. Che spesso vedono interrotto il neutro.

**Comment by cl-7k-01** [comment]
Grazie laspinacristian per la delucidazione! Questa è la risposta che cercavo!
